lib/presentation/widgets/logo_form.dart

```diff
@@ -32,18 +30,18 @@ class _MyLogFormWidgetState extends State<LogoForm> {
       key: formKey,
       child: Column(
         children: [
-          const Text(
-            greetings,
+          Text(
+            LocaleKeys.welcome_back_tony.tr(),
```

Comment:
@k1mka tr() method call is not needed everywhere. need to remove it from all places